streaming/src/main/scala/org/apache/spark/streaming/Checkpoint.scala

```diff
@@ -18,8 +18,8 @@
 package org.apache.spark.streaming
 
 import java.io._
-import java.util.concurrent.Executors
-import java.util.concurrent.RejectedExecutionException
+import java.util.concurrent.{ArrayBlockingQueue, RejectedExecutionException,
```

Comment:
Yeah it's the style I see elsewhere, like https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/SparkContext.scala#L39  Arguably it's time for a _ import at this stage; I'm indifferent here.